kategory/src/main/java/kategory/Convert.java

```diff
@@ -0,0 +1,55 @@
+package kategory;
+
+import org.jetbrains.annotations.NotNull;
+
+public class Convert {
+    @NotNull
+    public static <F, A> FromKindedJToKategory<F, A> fromKindedJ(@NotNull io.kindedj.HK<F, A> hk) {
```

Comment:
what's the point on creating a standard for HKs if you end up needing to translate from and to it ? Should the standard itself be the annotation processor capable of generating the HK abstractions over the types as we are doing now?